src/dictionary.js

```diff
@@ -0,0 +1,24 @@
+const https = require("https");
+const fs = require("fs");
+const generateList = async () => new Promise((resolve, reject) => {
+    let wordList = [];
+    if (fs.existsSync("../dictionary.json")) {
+        wordList = fs.readFileSync("../dictionary.json");
+        resolve(JSON.parse(wordList));
+    } else {
+        https.get("https://raw.githubusercontent.com/dwyl/english-words/master/words_alpha.txt", (res) => {
```

Comment:
Another task we can make is to have a separate config file where this dictionary URL is set

Reply:
I agree and spun up issue: https://github.com/prosif/homegames/issues/17
to track that